book.rb

```diff
@@ -0,0 +1,18 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
```

Comment:
- [ ] Kindly make the `rentals` array variable have a getter and setter method since we would need to update a `Book` or `Person` rental anytime that `Book` is rented or a `Person` wants to rent a book. 😄